doc/constitution.md

```diff
@@ -155,7 +155,7 @@ After the nomination deadline, the SC can prevent a nominee from becoming a cand
 
 #### Procedure
 
-The election is done using a [proportional representation mode from Condorcet Internet Voting Service (CIVS)](http://web.archive.org/web/20240412235900/https://civs1.civs.us/proportional.html). The recommendation is to use [CIVS](https://civs1.civs.us/) itself, but if CIVS is unavailable, a reasonable alternative proportional representation election implementation is also permitted.
+The election is done using a proportional representation oriented tallying system based on ranked ballots. One option is to use [proportional representation mode from Condorcet Internet Voting Service (CIVS)](http://web.archive.org/web/20240412235900/https://civs1.civs.us/proportional.html), but a reasonable alternative is also permitted.
```

Comment:
Not sure what the proper terminology is – Wikipedia calls it either [Electoral system](https://en.wikipedia.org/wiki/Proportional_representation) or [Voting system](https://en.wikipedia.org/wiki/Ranked_voting) (redirecting to https://en.wikipedia.org/wiki/Electoral_system). But tallying system is probably understandable enough.

Reply:
Good question; an ambiguity that is irrelevant for Wikipedia but relevant to us is that there is vote collection and vote tallying (and we already have experience where voting system for the vote collection did not have one of the promised properties for vote tallying).